screenpy/resolutions/is_empty.py

```diff
@@ -15,5 +16,9 @@ class IsEmpty(BaseResolution):
         the_actor.should(See.the(List.of_all(VIDEO_FRAMES), IsEmpty()))
     """
 
+    matcher: _IsEmpty
     line = "an empty collection"
     matcher_function = empty
+
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
I had to bypass pylint's useless-super-delegation rule to get this to commit. We should talk about how to handle pylint rules. (maybe use pylintrc file?)

Reply:
Yeah, i think we can handle those sorts of things better for sure.

On the other hand, do you actually need this `__init__` here? Maybe it has a point—does your IDE manage to typehint correctly with the things you added?

Otherwise we could add the `# pylint disable=useless-super-delegation` comment, with a small explanation.

Reply:
The IDE did not mark any sort of problem with a missing argument.
```python
the_actor.should(
            See.the(Number.of(ADVERTISEMENT_BANNERS), IsEqualTo())
```
I'd have to double check if `IsEmpty()` had the inverse problem.

Reply:
Oh jeez, you're totally right. I bet `IsEmpty` would definitely accept arguments, too.

Now i'm totally on board with these extra `__init__`s.